torchao/quantization/quant_primitives.py

```diff
@@ -944,10 +944,16 @@ def _choose_qparams_affine(
         else:
             zero_point = torch.full_like(scale, int((quant_max + quant_min + 1) / 2))
         scale = torch.clamp(scale, min=eps)
```

Comment:
should we modify `eps` to the right value instead of trying to clamp twice?  Right now `eps` is set to `torch.finfo(input.dtype).eps`, it seems like that just isn't the right way to set it here?